scripts/ci/forbidden-identifiers.js

```diff
@@ -16,15 +16,19 @@ var blocked_statements = [
 ];
 
 var blockedRegex = new RegExp('\\+.*(' + blocked_statements.join('|') + ').*$', 'mg');
+var commit_range = process.env.TRAVIS_COMMIT_RANGE || 'HEAD~1';
 
-var diff = child_process.execSync('git diff --unified=0 HEAD~1 ./src ./e2e').toString();
-var isInvalid = blockedRegex.test(diff);
+var diff = child_process.execSync(
+  'git diff --unified=0 ' + commit_range + ' ./src ./e2e'
```

Comment:
I can do that, but I don't think it's really necessary, because a plain string concatenation is doing the same. Plus additionally it's compatible with older node versions.